xslt/Image.xsl

```diff
@@ -38,18 +48,21 @@
 
     <!-- /ImageDocument/Metadata/Information/Image/Dimensions/S/Scenes/Scene => /OME/Image/Image -->
     <xsl:template match="Scene">
+        <xsl:variable name="idx" select="@Index"/>
```

Comment:
Is this in any way affected by a call to `AICSImage::set_scene`? So if I was to `set_scene(0)` in a multi-scene image then queried for acquisition time, I would (likely) get a different value than if I was `set_scene(5)`, then did the same?

Reply:
I'm not sure what all of the side-effects of `AICSImage::set_scene` are, but I don't think it would affect anything if you were querying against `AICSImage::ome_metadata`, since that property will always return the metadata for the entire file, rather than for a specific scene. Let me know if I'm misunderstanding your question, though.

Reply:
Mm, exactly what I was wondering. So is this loop var `idx` just an implementation detail of the translation mechanism? Not something that is set externally?

Reply:
Yes, exactly!